vexim/adminaliaschange.php

```diff
@@ -99,6 +108,9 @@
               } ?> class="textfield">
             </td>
           </tr>
+          <?php
+           if ($domrow['avscan'] == "1") {
+        ?>
```

Comment:
I wonder if this check (and the one checking if SA is enabled) is really necessary. It might be a good idea to set these values to something useful even when anti-virus or SA checks are disabled for a particular domain, don't you think so?